docs/source/handling_advanced_stypes/handle_text.rst

```diff
@@ -132,6 +156,197 @@ As mentioned earlier, :class:`~torch_frame.stype.text_embedded` is stored togeth
 Then, :obj:`stype_encoder_dict` can be directly fed into
 :class:`~torch_frame.nn.encoder.StypeWiseFeatureEncoder` to handle text columns.
 
+
+Finetuning Underlying Text Models
+---------------------------------
+
+For text columns with :class:`stype.text_tokenized<torch_frame.stype>`
+that finetunes underlying text models during the tabular learning, you need to specify both
+of the tokenization and encoding.
+Here, we use the
+`Transformers <https://huggingface.co/docs/transformers>`_ package.
+
+.. code-block:: bash
+
+    pip install transformers
+
+
+Specifying Text Tokenization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
shouldn't this subsection?

Reply:
This is a subsubsection which lies in the subsection
```
Using Pre-trained Text Embeddings
---------------------------------
```